common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -1198,6 +1198,12 @@
     ],
     "sqlState" : "54006"
   },
+  "EXCEED_ROW_LIMIT" : {
+    "message" : [
+      "Number of rows exceeds the allowed limit of <maxValues>. If this was intended, set <config> to at least the current row count."
```

Comment:
shall we mention transpose here?

Reply:
Cool mentioned in both name and message.